app/services/change_schedule.rb

```diff
@@ -111,6 +124,12 @@ def update_participant_profile_schedule_references!
   end
 
   def update_school_cohort_and_schedule!
+    changing_cohort = new_schedule.cohort != participant_profile.schedule.cohort
```

Comment:
In the current service (as opposed to the declaration service) we use `relevant_induction_record.schedule.cohort.start_year != cohort&.start_year` the induction record to determine the current cohort, I think we should probably do the same here to maintain consistency. For the record declaration the schedule on profile is used to avoid providers declaring against older cohorts. Let me know you're thoughts

Reply:
Yeah that makes sense, I'll update 👍